### PR TITLE
Doc: Add shared attribute for messaging ecs default info

### PR DESCRIPTION
**Pull request by karenzone**
## Release notes
[rn:skip] 

## What does this PR do?
Adds and links shared attribute file for accurately stating ecs default status.

### Design intent/usage
We can add/update text related to ecs defaults (or other info we want to write once and use multiple places). Then, we can  reference it by calling the attribute. In this case `{ecs-default}`.

**Important:** This work requires a corresponding change for the Versioned Plugin Reference. 

### To do (in this order): 
- [ ] Merge and backport this PR.
- [ ] Add shared attribute `include` file to the Versioned Plugin Reference guide, and test.
- [ ] Implement the attribute in plugin docs.  We can't start implementing the attribute in plugin docs until the framework is fully implemented. Otherwise, we will break the doc build.

### CODE REVIEW

**Comment by karenzone** [comment]
Hi @yaauie  I would appreciate your comments on the usefulness and content of this PR. 
Related: https://github.com/elastic/logstash-docs/pull/1128

File: docs/include/attributes-lsplugins.asciidoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/////
+ These settings control attributes in the LSR only. 
+They correspond to the VPR settings in logstash-docs/docs/versioned-plugins/include/attributes-ls-vpr.asciidoc
+When we update one, we must update settings in the other location
+/////
+
+:ecs-default:   Any value in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] for pipeline.ecs_compatibility other than disabled is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
```
Comment (yaauie) [inline docs/include/attributes-lsplugins.asciidoc:7]: nit: quote value

```suggestion
:ecs-default:   Any value in {logstash-ref}/logstash-settings-file.html[`logstash.yml`] for pipeline.ecs_compatibility other than `disabled` is considered BETA and unsupported until Logstash 8.0 and the final 7.x version are released. Having this pipeline level flag set will cause even patch-level upgrades to automatically consume breaking changes in the upgraded plugins, changing the shape of data the plugin produces.
```

Secondary: this setting can also be applied to an individual pipeline's definition in the `pipelines.yml`, and the BETA story is the same there.

File: docs/include/attributes-lsplugins.asciidoc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/////
+ These settings control attributes in the LSR only.
```
Comment (yaauie) [inline docs/include/attributes-lsplugins.asciidoc:2]: nit: extra whitespace

```suggestion
These settings control attributes in the LSR only. 
```

**Review by yaauie — APPROVED** [review]
👍🏼 LGTM

**Pull request merged**